File: src/main/java/com/mercadoeletronico/backendchallenge/dto/StatusRequestDTO.java
package com.mercadoeletronico.backendchallenge.dto;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class StatusRequestDTO {
	
	private String status;
	private Long itensAprovados;
	private Double valorAprovado;
	private String pedido;

}

File: ORDER.Domain/Dto/StatusRequestDto.cs
using System.Text.Json.Serialization;

namespace ORDER.Domain.Dto
{
    public class StatusRequestDto
    {
        [JsonPropertyName("status")]
        public string Status { get; set; }
        
        [JsonPropertyName("itensAprovados")]
        public int ApprovedItems { get; set; }
        
        [JsonPropertyName("valorAprovado")]
        public int ApprovedValue  { get; set; }
        
        [JsonPropertyName("pedido")]
        public string OrderId { get; set; }
    }
}